api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -408,6 +411,22 @@ public void unvoidVisit_shouldUnvoidEncountersVoidedWithVisit() {
 		assertEquals(1, encountersByVisit.size());
 	}
 	
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyListGivenNull() {
+		
+		assertThat(visitService.getVisitsByPatient(null), is(empty()));
+	}
+	
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyListGivenPatientWithNullPatientId() {
+		
+		Patient p = new Patient();
+		
```

Comment:
remove newline in between. these two lines go together, they form the arrange block which is needed as a preparation of the test.